pkg/acceptance/helpers/grant_client.go

```diff
@@ -129,6 +129,62 @@ func (c *GrantClient) RevokePrivilegesOnDatabaseFromDatabaseRole(
 	require.NoError(t, err)
 }
 
+func (c *GrantClient) GrantPrivilegesOnDatabaseToAccountRole(
```

Comment:
Nit: we reuse some code with `GrantOnSchemaToAccountRole` - it can be extracted.

Reply:
I guess you meant `GrantPrivilegesOnWarehouseToAccountRole`, merged them a bit